openmdao/recorders/case_reader_base.py

```diff
@@ -0,0 +1,66 @@
+from abc import ABCMeta, abstractmethod, abstractproperty
+
+
+class CaseReaderBase(object):
+    """ The Abstract base class of all CaseReader implementations. """
+
+    __metaclass__ = ABCMeta
+
+    def __init__(self, filename):
```

Comment:
no doctsring for the constructor?  I think we have been documenting the `__init__` Args in the class level docstring